src/install/kafka/windows/install-windows-boot.mdx

```diff
@@ -0,0 +1,48 @@
+---
+componentType: default
+headingText: Configure the Kafka integration
+---
+
+1. In the Integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`, create a copy of the sample configuration file by running:
+
+```
+copy kafka-config.yml.sample kafka-config.yml
+```
+
+2. Edit the `kafka-config.yml` file. The following example collects metrics and inventory discovered by the bootstrap broker. 
```

Comment:
Similar to the comment way earlier about expanding on the "Edit the file" part a bit.